frontends/web/src/components/inlineMessage/inlineMessage.tsx

```diff
@@ -14,25 +15,30 @@
  * limitations under the License.
  */
 
-import { Component } from 'react';
 import style from './InlineMessage.module.css';
 
-export default class InlineMessage extends Component {
-  deactivate = () => {
-    this.props.onEnd();
+type TProps = {
+    type: string;
+    message: string;
+    align: string;
+    onEnd: () => void;
+}
```

Comment:
I wanted to ask to also type type ('success') and align 'left' props, but this component is only used in tor proxy settings. It is probably not really worth it. I think when we update tor-proxy config we might even drop this component in favor or Message or Status component.